src/api.raml

```diff
@@ -191,6 +191,40 @@ traits:
               required: false
               example: [article/1234, interview/5678]
 
+/annotations:
+    (experimental):
+    type: collection
+    description: |
+        Annotations
+    get:
+        description: |
+            Get a list of annotations.
+        is:
+          - paged: { resources: Annotations }
+        queryParameters:
+            by[]:
+                description: |
+                    Only return annotations by at least one of these authors.
```

Comment:
As we're not going to be including author details, we should probably make this a single author, or it's a bit confusing?